internal/nodes/avi_model_nodes.go

```diff
@@ -768,7 +768,9 @@ func (v *AviHttpPolicySetNode) CalculateCheckSum() {
 	var checksum uint32
 	for _, hpp := range v.HppMap {
 		sort.Strings(hpp.Path)
-		checksum = checksum + utils.Hash(hpp.Host) + utils.Hash(utils.Stringify(hpp))
+		sort.Strings(hpp.Host)
+		checksum = checksum + utils.Hash(utils.Stringify(hpp))
```

Comment:
This will lead to checksum updates. I think there was a bug in the existing code since we were stringifying the `hpp` and we were also adding the Host field into the checksum which is already a part of the hpp.